templates/telemetry/dnssec-study-v1/dnssec-study-v1.4.schema.json

```diff
@@ -0,0 +1,212 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "dnssec-study-v1",
+  "properties": {
+    @TELEMETRY_ID_1_JSON@,
+    @TELEMETRY_CREATIONDATE_1_JSON@,
+    @TELEMETRY_APPLICATION_1_JSON@,
+    @TELEMETRY_ENVIRONMENT_1_JSON@,
+    @TELEMETRY_CLIENTID_1_JSON@,
+    "type": {
+      "type": "string",
+      "enum": [ "dnssec-study-v1" ]
+    },
+    "version": {
+      "type": "number",
+      "minimum": 4,
+      "maximum": 4
+    },
+    "payload": {
+      "type": "object",
+      "properties": {
+        "reason": {
+          "type": "string",
+          "enum": [ "startup", "measurementCompleted", "end",
```

Comment:
The enum might be unnecessarily strict -- if the reason doesn't conform here it will disappear into the error stream.